src/spacel/model/manifest.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, params={}):
                         for label, vol_params in
                         params.get('volumes', {}).items()}
         self.elb = params.get('elb')
+        self.elb_time = params.get('elb_time', 30)
```

Comment:
There is:
- Idle timeout: 60 seconds (for the connection draining)
- Healthcheck interval
- Healthcheck timeout
and a few thresholds

which make me think that, although this is a cool parameter to have, maybe we should keep in mind that this can turn out into an object. e.g. elb_healthcheck (with `timeout`, `interval`, `threshold` and other options) and e.g. elb_connection_draining_timeout